independent-projects/resteasy-reactive/server/vertx/src/test/java/org/jboss/resteasy/reactive/server/vertx/test/simple/MapWithParamConverterTest.java

```diff
@@ -0,0 +1,51 @@
+package org.jboss.resteasy.reactive.server.vertx.test.simple;
+
+import io.restassured.RestAssured;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import org.hamcrest.Matchers;
+import org.jboss.resteasy.reactive.RestQuery;
+import org.jboss.resteasy.reactive.server.vertx.test.framework.ResteasyReactiveUnitTest;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class MapWithParamConverterTest {
+
+	@RegisterExtension
+	static ResteasyReactiveUnitTest test = new ResteasyReactiveUnitTest()
+			.setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+					.addClasses(HelloResource.class, MapParamConverter.class, MapParamConverterProvider.class));
```

Comment:
Maybe add `MapParamConverter` and `MapParamConverterProvider` as static inner class of test as well as they are not going to be used by any other test